alpha-diversity.rmd

```diff
@@ -0,0 +1,154 @@
+---
+title: "alpha-diversity.rmd"
+output: html_document
+---
+
+```{r setup, echo=FALSE, results="asis"}
+library(rebook)
+chapterPreamble()
+```
```

Comment:
These setups are not in the beta diversity file. Is there a reason? Or can these be included there as well?